README.md

```diff
@@ -22,7 +22,7 @@ You can leverage this extension for your project in one of two ways:
 ## Applying the published extension
 
 The Common Custom User Data Maven extension is available in [Maven Central](https://search.maven.org/artifact/com.gradle/common-custom-user-data-maven-extension). This extension
-requires the [Develocity Maven extension](https://search.maven.org/artifact/com.gradle/gradle-enterprise-maven-extension) to also be applied in your build in order to have
+requires the [Develocity Maven extension](https://search.maven.org/artifact/com.gradle/develocity-maven-extension) to also be applied in your build in order to have
```

Comment:
Let's not make this change until the new maven extension is released.

Reply:
Done in https://github.com/gradle/common-custom-user-data-maven-extension/pull/202/commits/7c8b13f6c19b8007d0a64f0c95ce84b7b1fd8784